index.bs

```diff
@@ -375,11 +376,15 @@ different pages, or across navigations.
 <img alt="page's nonce" src="./resources/page-change-nonce.png"></img>
 
 *Storage and credentials created by anonymous iframes are no longer accessible
-after the top level frame navigated away because the Storage key for anonymous
-iframes will have changed. This applies to top-level history navigations as
-well, meaning that when the page navigates away, anything stored by anonymous
-iframes can be cleared by the browser, unless the page was stored in a
-back-forward cache.*
+after the top level frame navigated away toward a different top-level document,
+because the Storage key for anonymous iframes will be different. It means when
+the top-level document is released after a navigation, the storage used by
```

Comment:
```suggestion
because the Storage key for anonymous iframes will be different. It means if
the top-level document is destroyed after a navigation, the storage used by
```